src/dmqproto/client/request/Consume.d

```diff
@@ -205,10 +200,12 @@ public interface IController
     /***************************************************************************
 
         Tells the nodes to cleanly end the request.
+        Records that have already been received from the node will still be
+        passed to the user. It is possible to use `suspend` and `resume`.
```

Comment:
I'd say "In this situation, it is still possible to use `suspend` and `resume`."